src/Compilers/CSharp/Test/Emit3/PartialEventsAndConstructorsTests.cs

```diff
@@ -502,25 +502,25 @@ partial C() { }
             }
             """;
         CreateCompilation(source).VerifyDiagnostics(
-            // (4,33): error CS9277: Partial member 'C.E' may not have multiple defining declarations.
+            // (4,33): error CS9402: Partial member 'C.E' may not have multiple defining declarations.
             //     partial event System.Action E;
             Diagnostic(ErrorCode.ERR_PartialMemberDuplicateDefinition, "E").WithArguments("C.E").WithLocation(4, 33),
             // (4,33): error CS0102: The type 'C' already contains a definition for 'E'
             //     partial event System.Action E;
             Diagnostic(ErrorCode.ERR_DuplicateNameInClass, "E").WithArguments("C", "E").WithLocation(4, 33),
-            // (6,13): error CS9277: Partial member 'C.C()' may not have multiple defining declarations.
+            // (6,13): error CS9402: Partial member 'C.C()' may not have multiple defining declarations.
             //     partial C();
             Diagnostic(ErrorCode.ERR_PartialMemberDuplicateDefinition, "C").WithArguments("C.C()").WithLocation(6, 13),
             // (6,13): error CS0111: Type 'C' already defines a member called 'C' with the same parameter types
             //     partial C();
             Diagnostic(ErrorCode.ERR_MemberAlreadyExists, "C").WithArguments("C", "C").WithLocation(6, 13),
-            // (9,33): error CS9278: Partial member 'C.E' may not have multiple implementing declarations.
+            // (9,33): error CS9403: Partial member 'C.E' may not have multiple implementing declarations.
             //     partial event System.Action E { add { } remove { } }
             Diagnostic(ErrorCode.ERR_PartialMemberDuplicateImplementation, "E").WithArguments("C.E").WithLocation(9, 33),
             // (9,33): error CS0102: The type 'C' already contains a definition for 'E'
             //     partial event System.Action E { add { } remove { } }
             Diagnostic(ErrorCode.ERR_DuplicateNameInClass, "E").WithArguments("C", "E").WithLocation(9, 33),
-            // (11,13): error CS9278: Partial member 'C.C()' may not have multiple implementing declarations.
+            // (11,13): error CS9403: Partial member 'C.C()' may not have multiple implementing declarations.
             //     partial C() { }
             Diagnostic(ErrorCode.ERR_PartialMemberDuplicateImplementation, "C").WithArguments("C.C()").WithLocation(11, 13),
```

Comment:
>sealed [](http://example.com/codeflow?start=27&length=6)

It would be good to test situations when only one partial has `sealed` or `virtual` modifier. For other members too.

Reply:
Not sure what would you like to see tested in such scenarios. We already have test for the error that will occur "Both partial member declarations must have identical combinations of 'virtual', 'override', 'sealed', and 'new' modifiers." (That's tested for class, can extend that to interface if you want)